.github/workflows/cache_data.yaml

```diff
@@ -3,7 +3,7 @@ name: Cache data
 
 on:
   # Uncomment the 'pull_request' line below to manually re-cache data artifacts
-  # pull_request:
+  pull_request:
```

Comment:
```suggestion
  # pull_request:
```

Reply:
Is there something else that needs to be cached? I already uncommented/recommended this in a previous PR.

Reply:
What do you mean by "previous PR"?

The "Cache Data" workflow only needs to be run in scheduled jobs, not in PRs, so we need to comment this line (i.e., `# pull_request:`)